### PR TITLE
Fixed: Always true + Empty lines removed

### PR DESCRIPTION
**Pull request by ivandrofly**
/cc @niksedk

### CODE REVIEW

File: src/Logic/StripableText.cs [hunk 1]
```diff
@@ -42,7 +42,7 @@ public StripableText(string text, string stripStartCharacters, string stripEndCh
                     if (text.StartsWith("{\\", StringComparison.Ordinal))
                     {
                         int endIndex = text.IndexOf('}');
-                        if (endIndex > 0 && (!text.Contains('{') || text.IndexOf('{') > endIndex))
+                        if (endIndex > 0 && (!text.Substring(1).Contains('{') || text.IndexOf('{') > endIndex))
```
Comment (xylographe) [inline src/Logic/StripableText.cs]: Or, perhaps, slightly more efficient,
```
if (text.StartsWith("{\\", StringComparison.Ordinal))
{
    int endIndex = text.IndexOf('}', 2);
    if (endIndex > 0)
    {
        int index = text.IndexOf('{', 2);
        if (index < 0 || index > endIndex)
        {
            Pre += text.Substring(0, ++endIndex);
            text = text.Remove(0, endIndex);
        }
    }
}
```

Comment (ivandrofly) [inline src/Logic/StripableText.cs]: yeah but I always try to avoid the nested ```if```

Comment (xylographe) [inline src/Logic/StripableText.cs]: This one is also correct,
```
if (text.StartsWith("{\\", StringComparison.Ordinal))
{
    int endIndex = text.IndexOf('}', 2);
    if (endIndex > 0 && text.LastIndexOf('{', endIndex - 1) == 0)
    {
        Pre += text.Substring(0, ++endIndex);
        text = text.Remove(0, endIndex);
    }
}
```

**Comment by niksedk** [comment]
Committed in 6a48fa74cf09ff3e8c9a82d2e088e4f72c0bc3e

File: src/Logic/StripableText.cs (hunk 1 above)
Comment (ivandrofly) [inline src/Logic/StripableText.cs]: :+1:

**Pull request closed without merging**